reference/forms/types/number.rst

```diff
@@ -46,6 +47,14 @@ Field Options
 
 .. include:: /reference/forms/types/options/grouping.rst.inc
 
+html5
+~~~~~
+
```

Comment:
We're missing a `.. versionadded:: 4.3` directive

Reply:
`versionadded` directive added